package/mediacenter-addon-osmc/files/DEBIAN/postinst

```diff
@@ -41,13 +41,16 @@ then
 	if ! is_rbp; then rm -rf /usr/share/kodi/addons/script.module.osmcsetting.pi* >/dev/null 2>&1; fi
 	if is_atv; then rm -rf /usr/share/kodi/addons/script.module.osmcsetting.updates >/dev/null 2>&1; fi
 	if ischroot; then OWNER="1000:1000"; else OWNER="osmc:osmc"; fi
-	chown -R $OWNER /usr/share/kodi/addons/script.module.elementtree
```

Comment:
If we don't need / want script.module.elementree, then rip the chown and rm -rf line.